src/core/agent/services/identifierService.ts

```diff
@@ -359,6 +408,15 @@ class IdentifierService extends AgentService {
     delegateContact?: ConnectionShortDetails
   ): Promise<string | undefined> {
     const ourMetadata = await this.getMetadataById(ourIdentifier);
+    if (!ourMetadata.groupMetadata) {
+      throw new Error(IdentifierService.MISSING_GROUP_METADATA);
```

Comment:
You also need to check that `groupCreated` is false - so that this identifier can only be used for one multi-sig and not multiple